preferencesfx/src/main/java/com/dlsc/preferencesfx/history/view/HistoryTable.java

```diff
@@ -17,16 +17,12 @@
  */
 public class HistoryTable extends TableView<Change> {
 
-  ObservableList<Change> changes;
```

Comment:
as items is only associated with a weak reference to a `TableView`, the list could be garbage collected that way